src/pages/ProductList/MainPage/ProductCarousel.js

```diff
@@ -0,0 +1,100 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { Link } from 'react-router-dom';
+import './ProductCarousel.scss';
+
+const ProductCarousel = () => {
+  const TOTAL_PRODUCT_SLIDES = 3;
+  const slideRef = useRef(null);
+  const [currentIdx, setCurrentIdx] = useState(0);
+
+  useEffect(() => {
+    slideRef.current.style.transition = 'all 0.5s ease-in-out';
+    slideRef.current.style.transform = `translateX(-${currentIdx * 266}px)`;
+  }, [currentIdx]);
+
+  const nextProduct = () => {
+    if (currentIdx >= TOTAL_PRODUCT_SLIDES) {
+      setCurrentIdx(0);
+    } else {
+      setCurrentIdx(prev => prev + 1);
+    }
+  };
+
+  const prevProduct = () => {
+    if (currentIdx === 0) {
+      setCurrentIdx(TOTAL_PRODUCT_SLIDES);
+    } else {
+      setCurrentIdx(prev => prev - 1);
+    }
+  };
+
+  return (
+    <div className="mainProductList">
+      <h1 className="productTitle">
+        가격 거품을 걷어내고, 성분에 집중한 스킨케어
+      </h1>
+      <div className="productCarousel">
+        <div className="productCarouselBox " ref={slideRef}>
+          {PRODUCT_INFO.map(cardElement => (
+            <ProductCard key={cardElement.id} card={cardElement} />
+          ))}
```

Comment:
보통 array의 경우 복수형으로 작성하고 콜백의 인자는 단수로 작성합니다.
```suggestion
          {PRODUCTS.map(product => (
            <ProductCard key={product.id} card={product} />
          ))}
```